octobot_trading/exchanges/exchange_builder.py

```diff
@@ -82,7 +82,6 @@ async def _build_trader(self):
 
     async def _build_modes(self):
         try:
-            init_trading_mode_config(self.config)
```

Comment:
Managed by tentacle manager ?

Reply:
By OctoBot, it's related to the advanced manager (when we figure out advanced classes). Maybe we should move it into the tentacle manager if we decide it's its role to manage advanced classes in the future

Reply:
Yes it should be better in Tentacle manager, I don't think Octobot should do that.

Reply:
related to https://github.com/Drakkar-Software/OctoBot-Tentacles-Manager/issues/11